Libs/DICOM/Core/ctkDICOMDatabase.cpp

```diff
@@ -732,6 +732,23 @@ QStringList ctkDICOMDatabase::seriesForStudy(QString studyUID)
 }
 
 //------------------------------------------------------------------------------
+QStringList ctkDICOMDatabase::sopsForSeries(const QString seriesUID)
```

Comment:
Let's call this instancesForSeries to be more consistent with other methods like fileForInstance and fileForInstance.

Otherwise looks great.